keras/backend/mxnet_backend.py

```diff
@@ -5706,15 +5715,15 @@ def set_mxnet_context(self, context):
                                      'call `multi_gpu_model` with `len(gpus) >= 2`. '
                                      'Received: `gpus=%s`' % context)
             elif isinstance(context, str):
-                if not context.lower().startswith('eia'):
+                if not context.lower().startswith(('eia', 'cpu', 'gpu')):
                     raise ValueError('MXNet Backend: Invalid context provided - %s' % context)
             else:
                 if context <= 1:
                     raise ValueError('MXNet Backend: For multi-gpu usage to be effective, '
                                      'call `multi_gpu_model` with `gpus >= 2`. '
                                      'Received: `gpus=%d`' % context)
 
-            self._context = _get_mxnet_context(context)
+            self._context = self.get_mxnet_context(context)
```

Comment:
why move inside the class? Any specific reason?

Reply:
reverted back